extractors/personal_identifiers/location_extraction/code_snippet_common.md

```diff
@@ -0,0 +1,43 @@
+```python
+import spacy
+from typing import List, Tuple
+
+loaded_models = {}
+def load_spacy(spacy_model):
+    if spacy_model not in loaded_models:  
+        loaded_models[spacy_model] = spacy.load(spacy_model)
+    return loaded_models[spacy_model]
+
+
+def location_extraction(text: str, extraction_keyword: str, spacy_model: str = "en_core_web_sm") -> List[Tuple[str, int]]:
```

Comment:
return type needs to be `List[Tuple[str, int, int]]`

Reply:
Done. :)